examples/eks-msk-opensearch/README.md

```diff
@@ -0,0 +1,102 @@
+## Multi-Tenant logging using Fluent-bit, EKS , Amazon MSK and  Amazon OpenSearch
+
+In this example, we will showcase how to send your application logs from EKS to OpenSearch via Amazon MSK (managed KAFKA). We will use tenant notion using EKS "namespace" such that each tenant has a seperate namespace.  The solution helps to meet following use cases in a multi-tenant deployment from EKS cluster.
```

Comment:
>  We will use tenant notion using EKS "namespace" such that each tenant has a seperate namespace.

Sorry this sentence does not make sense to me. I think it may be need to expanded. I think here we need to briefly introduce what a tenant is and why the customer cares. Also if the "namespace" here is just kubernetes namespaces I think it would be more clear to just use that phrase, and that it doesn't need to be in quotes. 

Also I always get this one wrong too but its "separate".